zebrad/src/components/inbound/cached_peer_addr_response.rs

```diff
@@ -0,0 +1,96 @@
+//! Periodically-refreshed GetAddr response for the inbound service.
+//!
+//! Used to avoid giving out Zebra's entire address book over a short duration.
+
+use std::{
+    sync::{Mutex, TryLockError},
+    time::Instant,
+};
+
+use super::*;
+
+/// The maximum duration that a `CachedPeerAddrResponse` is considered fresh before the inbound service
+/// should get new peer addresses from the address book to send as a `GetAddr` response.
+pub const INBOUND_CACHED_ADDRS_REFRESH_INTERVAL: Duration = Duration::from_secs(10 * 60);
+
+/// The maximum duration that a `CachedPeerAddrResponse` is considered fresh before the inbound service
```

Comment:
These docs have been copied and pasted, so they aren't accurate for this constant.

Reply:
Fixed in https://github.com/ZcashFoundation/zebra/pull/7955/commits/a4a8963c70058a16fa8502d1703e431c0f05141b.